Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -3766,8 +3766,26 @@ script:
       name: device-group
     - description: Return raw XML
       name: is_xml
+    - name: is_file
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: Return response as file.
+      defaultValue: "false"
+    - name: file_name
+      description: Optional file name when running this command with is_file=true.
+      defaultValue: pan-os-result-file
```

Comment:
default value is a command? why?

Reply:
Sorry did not mean to make it look like a command! The idea is just to provide a generic filename if one is not supplied. Will change it.